CMakeLists.txt

```diff
@@ -325,6 +325,16 @@ if(ARCH_ID STREQUAL "s390x")
   set(S390X 1)
 endif()
 
+if(ARCH_ID STREQUAL "riscv64")
+set(RISCV   1)
+set(RISCV64 1)
+endif()
+
+if(ARCH_ID STREQUAL "riscv32")
+set(RISCV   1)
+set(RISCV32 1)
```

Comment:
Is this variable and its 64 variant used somewhere or did you add it for possible future use?

Reply:
@mj-xmr For possible future use. If you think they are now redundant, please tell me and I will remove them.

Reply:
IMO it's fine to keep it

Reply:
I was asking mainly if I missed something in other part of the file.

It looks reasonable to have this.